### ISSUE TITLE
aria-describedby: pass when one ID in a set of multiple values can't be found

### ISSUE THREAD

**Issue by marcysutton** [dequelabs/axe-core#876]
When `aria-describedby` points to multiple IDREFs, axe-core will fail if any of the referenced elements can't be found. It makes sense to fail if none can be found, but if at least one is found, it gets read aloud in every AT I checked (Voiceover OSX and iOS, NVDA and JAWS). It seems like something we could change since it doesn't negatively impact users.

**Comment by marcysutton** [comment]
Ah https://github.com/dequelabs/axe-core/issues/875 was opened before I finished typing, so I'll close this as a duplicate.
